azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/middleware/sdk/ChatServiceListener.kt

```diff
@@ -87,7 +88,17 @@ internal class ChatServiceListener(
             is MessageInfoModel -> {
                 when (it.eventType) {
                     ChatEventType.CHAT_MESSAGE_RECEIVED -> {
-                        dispatch(ChatAction.MessageReceived(message = it.infoModel))
+                        val infoModel = it.infoModel
+                        dispatch(ChatAction.MessageReceived(message = infoModel))
+                        it.infoModel.senderCommunicationIdentifier?.let {
+                            dispatch(
+                                ParticipantAction.RemoveParticipantTyping(
```

Comment:
Parameter should probably just be user id

Reply:
We need both id+timestamp to identify unique notification